geode-core/src/integrationTest/java/org/apache/geode/cache/client/internal/AutoConnectionSourceImplJUnitTest.java

```diff
@@ -420,6 +420,56 @@ public void testLocatorUpdateIntervalZero() {
     assertEquals(updateLocatorInterval, source.getLocatorUpdateInterval());
   }
 
+  /**
+   * This tests validates update of IP of locator from initialLocators list.
+   * In this test we validate this using identityHashCode.
+   */
+  @Test
+  public void testinitialLocatorIpChange() {
+    int port = 11011;
+    List<InetSocketAddress> locators = new ArrayList<>();
+    InetSocketAddress floc1 = InetSocketAddress.createUnresolved("localhost", port);
+    InetSocketAddress floc2 = new InetSocketAddress("fakeLocalHost2", port);
+
+    locators.add(floc1);
```

Comment:
`locators` is actually never used, values are only added to this list and never accessed